de.dlr.sc.virsat.model.extension.fdir.ui/src/de/dlr/sc/virsat/model/extension/fdir/ui/diagram/ft/features/faultTreeNodes/FaultTreeNodeLayoutFeature.java

```diff
@@ -47,16 +48,22 @@
 
 	public static final int PADDING_X = 10;
 	public static final int PADDING_Y = 10;
-	
+
 	public static final double SPARE_RELATIVE_TYPE_Y = 2d / 3;
 	public static final int SPARE_RECT_POS_Y = 12;
 	public static final double SPARE_RECT_REL_POS_X = 0.5;
-	
+
+	private static final String TRUE = "true";
+	private static final String IS_COMMENT = "is-comment";
+	private static final String BELONGS_TO_FAULT_NODE = "Belongs-to-fault-node";

```

Comment:
For consistency I think we mainly use uncaptialized letters for the properties, so we should do that here as well.